External.LCA_RESTRICTED/Languages/IronPython/27/Lib/test/test_mmap.py

```diff
@@ -543,6 +546,7 @@ def __new__(klass, *args, **kwargs):
         anon_mmap(PAGESIZE)
 
     @unittest.skipUnless(hasattr(mmap, 'PROT_READ'), "needs mmap.PROT_READ")
+    @unittest.skipIf(sys.platform=='cli' and os.name=='posix', 'IronPython does not support this yet')
```

Comment:
I wonder if we should include issue numbers for these types of exclusion, would make finding things these later down the line.